plugins/outputs/azure_data_explorer/azure_data_explorer.go

```diff
@@ -103,9 +109,9 @@ func (adx *AzureDataExplorer) Write(metrics []telegraf.Metric) error {
 func (adx *AzureDataExplorer) writeTablePerMetric(metrics []telegraf.Metric) error {
 	tableMetricGroups := make(map[string][]byte)
 	// Group metrics by name and serialize them
-	for _, m := range metrics {
-		tableName := m.Name()
-		metricInBytes, err := adx.serializer.Serialize(m)
+	for _, metric := range metrics {
+		tableName := metric.Name()
+		metricInBytes, err := adx.serializer.Serialize(metric)
```

Comment:
Please keep the original code as the new code will likely mask the `metric` include and lead to linter issues.